### PR TITLE
Add ObjectLoader, Errata, Tidy

### PR DESCRIPTION
**Pull request by mikegoodstadt**
Issues #3 and #4.
Add ObjectLoader for viewing THREEjs Object JSON files.

### CODE REVIEW

File: js/threeView.js [hunk 1]
```diff
@@ -1,291 +0,0 @@
...
-                        }
-                    }
-                }
-            }
-        }
-    }
-
-    window.Viewer = webglSupport() ? Viewer() : noWebGLFallback();
-
-})();
```
Comment (OmriAharon) [inline js/threeView.js:291]: Good that this is deleted. Can you please also add this file to .gitignore?

File: js/viewer.js [hunk 2]
```diff
@@ -281,6 +306,12 @@ var Viewer = function () {
 
     this.loadObject = function (obj, type, onViewerReady, onLoadError) {
         switch (type) {
+            case 'js':
+                $this.loadJson(obj, onViewerReady, onLoadError);
+                break;
+            case 'json':
+                $this.loadJson(obj, onViewerReady, onLoadError);
+                break;
```
Comment (OmriAharon) [inline js/viewer.js:314]: Can be merged to:
```
case 'js':
case 'json':
        $this.loadJson(obj, onViewerReady, onLoadError);
        break;
```

File: dist/three-d-viewer.html [hunk 3]
```diff
@@ -402,6 +427,12 @@
 
     this.loadObject = function (obj, type, onViewerReady, onLoadError) {
         switch (type) {
+            case 'js':
+                $this.loadJson(obj, onViewerReady, onLoadError);
+                break;
+            case 'json':
+                $this.loadJson(obj, onViewerReady, onLoadError);
+                break;
```
Comment (OmriAharon) [inline dist/three-d-viewer.html:435]: Can be merged to:

```
case 'js':
case 'json':
        $this.loadJson(obj, onViewerReady, onLoadError);
        break;
```

**Review by OmriAharon — APPROVED** [review]
Thanks for the PR!

**Comment by OmriAharon** [comment]
Also, since you added a json loader, please update README file that it is also supported in the file type line.

**Comment by mikegoodstadt** [comment]
Reverted b40525d and 0d61bf1 as they are internal to our own work and were not meant as general updates for Autodesk's 3DViewerComponent.

**Comment by mikegoodstadt** [comment]
README updated

**Pull request merged**